clusterloader2/pkg/measurement/common/simple/service_creation_latency.go

```diff
@@ -0,0 +1,538 @@
+/*
```

Comment:
This confirms my initial claim months ago, that this directory shouldn't have been called "simple/" :D

Reply:
Done.